source/dotnet/Library/AdaptiveCards/AdaptiveTypedBaseElementConverter.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
Nit: Should be able to clean up the using's here to remove ones not being used in the file

Reply:
thanks

---
In reply to: [341278562](https://github.com/microsoft/AdaptiveCards/pull/3548#discussion_r341278562) [](ancestors = 341278562)